UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/Helper/FileContentHelper.cs

```diff
@@ -0,0 +1,277 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using UKHO.ExchangeSetService.API.FunctionalTests.Models;
+
+namespace UKHO.ExchangeSetService.API.FunctionalTests.Helper
+{
+    public static class FileContentHelper
+    {
+        private static FssApiClient FssApiClient = new FssApiClient();
+
+        public static void CheckSerialEncFileContent(string inputfile)
+        {
+            string[] lines = File.ReadAllLines(inputfile);
+
+            //Store file content
+            string[] fileContent = lines[0].Split(" ");
+
+            string dataServerAndWeek = fileContent[0];
+            string dateAndCdType = fileContent[3];
+            string formatVersionAndExchangeSetNumber = fileContent[7];
+
+            int weekNumber = CultureInfo.InvariantCulture.Calendar.GetWeekOfYear(DateTime.UtcNow, CalendarWeekRule.FirstFullWeek, DayOfWeek.Thursday);
+            string year = DateTime.UtcNow.Year.ToString().Substring(DateTime.UtcNow.Year.ToString().Length - 2);
+            string currentDate = DateTime.UtcNow.ToString("yyyyMMdd");
+
+            Assert.AreEqual(dataServerAndWeek, $"GBWK{weekNumber}-{year}");
+
+            Assert.AreEqual(dateAndCdType, $"{currentDate}UPDATE");
+
+            Assert.IsTrue(formatVersionAndExchangeSetNumber.StartsWith("02.00U01X01"), $"Expected format version {formatVersionAndExchangeSetNumber}");
+
+        }
+
+        public static void CheckProductFileContent(string inputfile, dynamic scsResponse, string ScsJwtToken)
```

Comment:
Remove unused parameter string ScsJwtToken . Ensure camel casing is followed for parameter and variable names across all methods of FileContentHelper.cs file

Reply:
updated